reflex/compiler/compiler.py

```diff
@@ -98,11 +98,15 @@ def _compile_contexts(state: Type[State]) -> str:
     Returns:
         The compiled context file.
     """
-    return templates.CONTEXT.render(
-        initial_state=utils.compile_state(state),
-        state_name=state.get_name(),
-        client_storage=utils.compile_client_storage(state),
-        is_dev_mode=os.environ.get("REFLEX_ENV_MODE", "dev") == "dev",
+    return (
+        templates.CONTEXT.render(
+            initial_state=utils.compile_state(state),
+            state_name=state.get_name(),
+            client_storage=utils.compile_client_storage(state),
+            is_dev_mode=os.environ.get("REFLEX_ENV_MODE", "dev") == "dev",
+        )
+        if state
+        else templates.CONTEXT.render()
```

Comment:
`is_dev_mode` can probably still be rendered yeah?

Reply:
This renders as `null` if there's no state, Im guessing `is_dev_mode`  is required regardless of whether state is used or not

Reply:
`is_dev_mode` isn't _really_ required at all, it's just an extra var that can be imported to determine if reflex is running in dev or prod mode. The value can be determined regardless of state, as it only depends on checking an env var at compile time.